<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winBank</class>
 <widget class="QDialog" name="winBank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>520</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nouvelle Banque</string>
  </property>
  <widget class="QLabel" name="labelBank">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../ressources/ressources.qrc">:/winPictures/bank_128.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelDesc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Les Etablissements Bancaires&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Dans ce formulaire vous pouvez donner des détails sur vos établissements&lt;br/&gt;bancaires. Cela peux être utilise de pouvoir envoyer un mail ou téléphoner a&lt;br/&gt;sa banque ou personnaliser vos rapports&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelBankName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de l'établissement bancaire :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBankAdress">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse de l'établissement bancaire :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBankName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBankAdr1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBankAdr2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBankAdr3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelBankPhone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Téléphone de la banque :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBankMail">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mail de la banque :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBankPhone">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBankMail">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxContactBank">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Contact à la banque :</string>
   </property>
   <widget class="QLabel" name="labelContactName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nom et prénom du contact :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditContactName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelContactPhone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Téléphone :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelContactMail">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mail :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditContactPhone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditContactMail">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushBtnOK">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBank">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>471</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnSupp">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
